pkg/common/util/webhooks.go

```diff
@@ -0,0 +1,23 @@
+package util
+
+import (
+	v1 "github.com/kubeflow/training-operator/pkg/apis/kubeflow.org/v1"
+
+	"k8s.io/apimachinery/pkg/util/sets"
+	"k8s.io/apimachinery/pkg/util/validation/field"
+)
+
+var SupportedJobControllers = sets.New(
```

Comment:
```suggestion
var supportedJobControllers = sets.New(
```

Can we avoid exposing this variable?

Reply:
+1